rules/allied-infantry.yaml

```diff
@@ -106,26 +108,38 @@ e1:
 		Weapon: M60
 		UpgradeTypes: eliteweapon
 		UpgradeMaxEnabledLevel: 0
-		UpgradeMaxAcceptedLevel: 1
+		UpgradeMaxAcceptedLevel: 0
```

Comment:
This is nonsense and prevents eliteweapons being used entirely. This should be 3 and should be listed on all weapons.